modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -101,7 +101,16 @@ If any of these conditions are true, do not force wipe the disk. Instead, you mu
 
 |`deviceClasses.thinPoolConfig`
 |`object`
-|Contains the configuration to create a thin pool in the LVM volume group.
+|Contains the configuration to create a thin pool in the LVM volume group. 
+
+If you exclude this field, logical volumes are thick-provisioned. 
+
+The limitations of using thick-provisioned storage are as follows:
+
+* Lack of copy-on-write support for volume cloning.
+* Lack of support for snapshot class.
+* Lack of support for over-provisioning. As a result, the provisioned capacity of `PersistentVolumeClaims` (PVCs) is immediately reduced from the volume group.
+* Thick-provisioned devices lack support for thin metrics. They only support volume group metrics.
```

Comment:
```suggestion
* No support for thin metrics. Thick-provisioned devices only support volume group metrics.
```

Reply:
I think that these changes make the list a little more succinct and parallel without changing the meaning too much.